packages/channels/botpress-channel-messenger/config.json

```diff
@@ -16,6 +16,7 @@
 
   "graphVersion": "2.12",
   "enabled": true,
+  "enableAllProfileFields": false, // Before fetch all profile fields, you need to submit all FB reviews
```

Comment:
no need for comma here:
`To be able to fetch all profile fields you need to submit your application for FB review`